sphinx_gallery/utils.py

```diff
@@ -175,3 +175,12 @@ def _has_pypandoc():
         return None, None
     else:
         return True, version
+
+
+def _has_graphviz():
+    try:
+        import graphviz  # noqa F401
+    except ImportError:
+        logger.info('`graphviz` required for graphical visualization')
```

Comment:
It says this here, should it say something else?

Reply:
The `f'` and `{exc}` in my comment were intended as hints toward this more fleshed out version
```suggestion
        logger.info('`graphviz` required for graphical visualization '
                    f'but could not be imported, got: {exc}')
```